testing/environments/snapshot.yml

```diff
@@ -21,4 +25,6 @@ services:
     build:
       context: ./docker/kibana
       dockerfile: Dockerfile-snapshot
-
+      args: [ ]
+        #KIBANA_DOWNLOAD_URL: https://snapshots.elastic.co/downloads/kibana/kibana-6.0.0-alpha1-SNAPSHOT-linux-x86_64.tar.gz
```

Comment:
This is left out currently as it throws errors in the inherited compose files in the tests which don't require the variables. Variables were left in as "comment"